<?xml version="1.0" encoding="UTF-8"?>
<!--
 Copyright (C) 2010 Bruneau Basile
 
 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 2 of the License, or
 (at your option) any later version.
 
 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License along
 with this program; if not, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-->
<ui version="4.0">
 <class>preFen</class>
 <widget class="QMainWindow" name="preFen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COW</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>cow-v0.8-transparent.png</normaloff>cow-v0.8-transparent.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-image: url(:/fond2.png);
background-repeat: no-repeat;
background-color: #c9d0d8;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="pixmap">
           <pixmap resource="cow.qrc">:/cow-v0.8-transparent.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="la_a">
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;px.png&quot;);
border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;: Choose a nickname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="t_pseudo">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
border: 2px solid black;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="la_b">
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;px.png&quot;);
border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;: Search your school&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="t_serveur_recherche">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
border: 2px solid black;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="t_serveur_liste">
          <property name="styleSheet">
           <string notr="true">QListView {
border: 2px solid black;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="chargement">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="a_ten">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ajouter_etab">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }</string>
          </property>
          <property name="title">
           <string>Add a school</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="ajjj">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="aj_nom">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>City:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="aj_ville">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>ZIP code:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="aj_codep">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Description:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="aj_description">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background: white;
selection-background-color: darkgray;
}</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Type:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="aj_type">
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
     border: 1px solid gray;
     border-radius: 3px;

     min-width: 6em;
 }</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>College</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>High school</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Other</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Are you actually in this school?</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="aj_dedans">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QCheckBox {
     spacing: 5px;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
 }</string>
                 </property>
                 <property name="text">
                  <string>Yes</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="aj_envoyer">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder></zorder>
          <zorder>aj_envoyer</zorder>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="t_ips">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Server's IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="t_ports">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>10100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="la_c">
          <property name="styleSheet">
           <string notr="true">background-image: url(&quot;px.png&quot;);
border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;: Get connected!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_connexion">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
     border: 2px solid black;
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }</string>
          </property>
          <property name="text">
           <string extracomment="Il s'agit du bouton de connexion">CONNEXION</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="cow.qrc"/>
 </resources>
 <connections/>
</ui>
